www/js/services/incidents_services.js

```diff
@@ -12,10 +12,10 @@ angular.module('mobility')
     area: "KAKAAKO",
     lat: 21.297355,
     lng: -157.861581,
-    icon: '../../img/ionic.png'
+    icon: 'img/ionic.png'
```

Comment:
Right!  We don't need it.  I forgot to remove it.  I'll do so in a later pull request.